core/src/components/select-popover/select-popover.tsx

```diff
@@ -116,19 +116,28 @@ export class SelectPopover implements ComponentInterface {
 
   renderCheckboxOptions(options: SelectPopoverOption[]) {
     return options.map((option) => (
-      <ion-item class={getClassMap(option.cssClass)}>
+      <ion-item
+        class={{
+          // TODO FW-4784
```

Comment:
In the future we will be able to use `:has` instead, but it's still too new to use in Ionic: https://caniuse.com/?search=%3Ahas